.github/workflows/code-analysis.yml

```diff
@@ -20,7 +20,6 @@ name: Code analysis
 on:
   workflow_call:
   pull_request:
-  push:
```

Comment:
I recommend keeping these when pushes to branch.  It helps to generate a mark that the master is clean.

Reply:
This does not affect release because it's only marked on "branch" but not "tags"

Reply:
It does generate that mark. But no one is paying attention to the test results triggered by push.
And the PRs are already passing the check based on the latest code, I don't think this is necessary.